pkg/kritis/cryptolib/verifier_test.go

```diff
@@ -19,6 +19,8 @@ package cryptolib
 import (
 	"testing"
 
+	"github.com/grafeas/kritis/pkg/kritis/testutil"
```

Comment:
I would be extra cautious of making another kritis library as dependency. It makes the crypto library less self-contained, and also can complicate porting the library to other places.

Reply:
Done. Removed dependency